test/blackbox-tests/test-cases/melange/multilib.t/run.t

```diff
@@ -0,0 +1,16 @@
+Compilation using melange with a library
+The rules should include cmjs of dependencies as well.
+
+  $ dune build x/.x.objs/melange/x__M_2.js
+
+Make sure no byte folders are included.
+
+  $ dune rules x/.x.objs/melange/x__M_2.js  | tr -s '\n' ' ' |
+  > grep -ce "byte"
+  0
+  [1]
```

Comment:
Instead of checking the output of `dune rules` (which is somewhat an internal detail), I would use the output of  `dune build --display=short` in the CRAM test.

Reply:
@nojb thanks for the feedback.

I am not sure I understand how `dune build --display=short` would work to test that no `byte` paths are passed to the `-I` flags when building melange libraries with `melc`. As far as I can see, `dune build --display=short` shows the targets of the build, but not the dependencies or the commands used to generate said targets. Maybe there is a way to print so with `dune build`?